galasa-ui/src/tests/components/PageHeaderMenu.test.tsx

```diff
@@ -158,6 +158,23 @@ test('clicking log out button calls handleDeleteCookieApiOperation, RESPONSE OK'
 
 });
 
+// Mock matchMedia
+beforeAll(() => {
+  Object.defineProperty(window, 'matchMedia', {
+    writable: true,
+    value: jest.fn().mockImplementation(query => ({
+      matches: false, // or true, depending on your test case
+      media: query,
+      onchange: null,
+      addListener: jest.fn(), // deprecated
+      removeListener: jest.fn(), // deprecated
+      addEventListener: jest.fn(), // modern
+      removeEventListener: jest.fn(),
+      dispatchEvent: jest.fn(),
```

Comment:
Do we need to define all of these functions in this mock?

We could only implement the methods/properties that we use perhaps?